content/docs/self-hosted-runners.md

```diff
@@ -202,16 +202,23 @@ env:
 
 ## Using on-premise machines as self-hosted runners
 
-You can also use the new `cml-runner` function to set up a local self-hosted
-runner. On your local machine or on-premise GPU cluster, you'll install CML as a
-package and then run:
+You can also use `cml-runner` to set up a self-hosted runner on your local
+machine or on-premise GPU cluster.
 
 ```dvc
 cml-runner \
-  --repo $your_project_repository_url \
-  --token=$personal_access_token \
-  --labels tf \
-  --idle-timeout 180
+  --repo="$repository_url" \
+  --token="$personal_access_token" \
+  --labels="local,runner" \
```

Comment:
I do not see the need of runner tag

Reply:
I added it to illustrate the use of comma-separated tags with no spaces between them, but naming the second tag `runner` is really redundant.

Reply:
Feel free to create a follow-up pull request removing it.